src/sentry/static/sentry/app/components/charts/components/types.tsx

```diff
@@ -0,0 +1,29 @@
+/**
+ * eCharts markLine
+ *
+ * See https://echarts.apache.org/en/option.html#series-line.markLine
+ */
+
+type FormatterCallback = (any) => string;
+
+type StyleDraft = {
+  show?: boolean;
+  position?: string;
+  distance?: number | number[];
+  formatter?: string | FormatterCallback;
+};
+
+type LineStyle = StyleDraft & {
+  emphasis?: StyleDraft;
+};
+
+export type MarkLine = {
```

Comment:
These types might already be exported from echarts I think, so we don't have to duplicate them.

Reply:
It isn't exported directly, it is a part of some of the exported types from echarts but it isn't in the namespace only in the non-exported interface